packages/lib/src/components/Boleto/types.ts

```diff
@@ -30,3 +30,9 @@ export interface BoletoInputErrorState {
     shopperEmail?: boolean;
     socialSecurityNumber?: boolean;
 }
+
+// An interface for the members exposed by a component to its parent UIElement
+export interface BoletoRef {
+    showValidation?: (who) => void;
+    setStatus?: any;
```

Comment:
As this is a pattern that applies to most (if not all) of Preact Components that the UIElement interacts directly, perhaps we could try to create an interface and reuse it across different Components. 

Ex:

```ts
interface ComponentMethodsRef {
   showValidation?(...
   setStatus?(...
   ...
}
```

```jsx
const boletoRef = useRef<ComponentMethodsRef>({});
```

Maybe I'm  overengineering here 😅  But this pattern is very repetitive. It feels like can be improved somehow

Reply:
Yup, good idea!